bespin_web/tasks/loaddata.yml

```diff
@@ -28,11 +27,10 @@
 - name: Specific Load Commands
   docker_container:
     command: "python manage.py {{ item }}"
-    name: bespin-load-command-runner
+    name: bespin-loaddata
     image: dukegcb/bespin-web
     pull: true
     state: started
-    auto_remove: true
```

Comment:
I think we'd want to set `detach` to false here, so that the process stays in the foreground and ansible can tell if it fails. Also, it seems from the [docs](http://docs.ansible.com/ansible/latest/docker_container_module.html#docker-container), that if the container is detached, then the next step could destroy it before it completes.

We had a similar situation with [imads](https://github.com/Duke-GCB/gcb-ansible-roles/blob/master/imads_webapp/tasks/setup_postgres.yml), where we do set `detach` to false, but also ran a `docker wait` shell command to wait for the load process to finish. 

`auto_remove` looks like it would help cleanup too, but I don't know if we have 2.4.